src/lib/evil/evil_time.c

```diff
@@ -3,6 +3,7 @@
 #endif /* HAVE_CONFIG_H */
 
 #include "evil_string.h"
+#include "evil_time.h"
```

Comment:
When we're including things from evil, always include `evil_private.h` :) It is a weird default from EFL, but...it is the way it works haha.

```suggestion
#include "evil_private.h"
```

Also, ensure it is the last included header.

Reply:
@JPTIZ , desfiz o commit porque evil_private.h já estava incluído

Reply:
OOh, tendi :) Beleza.

Reply:
It's still here though.